src/gtnh/utils.py

```diff
@@ -91,3 +97,24 @@ def get_mod_asset(release: GitRelease) -> GitReleaseAsset:
             continue
 
         return asset
+
+
+def get_maven(mod_name: str) -> Optional[str]:
+    maven_url = MAVEN_BASE_URL + mod_name + "/"
+    response = requests.head(maven_url)
+
+    return maven_url if response.status_code == 200 else None
```

Comment:
could be cool to log a warn if the status code is a 5xx to let the user know it's not missing but unreachable